tests/queries/0_stateless/03381_udf_asterisk.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+FUNCTION_NAME="function_$CLICKHOUSE_DATABASE"
+
+$CLICKHOUSE_CLIENT -q "
+  DROP FUNCTION IF EXISTS $FUNCTION_NAME;
+  DROP TABLE IF EXISTS t;
+"
+
+$CLICKHOUSE_CLIENT -q "
+  CREATE FUNCTION $FUNCTION_NAME AS (x) -> (x AS c0);
+  CREATE TABLE t (t0 UInt64, t1 UInt64) ENGINE=MergeTree() ORDER BY t0;
+"
+
+$CLICKHOUSE_CLIENT -q "CREATE VIEW x AS (SELECT $FUNCTION_NAME(*)); -- { serverError BAD_ARGUMENTS }"
+$CLICKHOUSE_CLIENT -q "CREATE VIEW x AS (SELECT $FUNCTION_NAME(t.*) FROM t); -- { serverError BAD_ARGUMENTS }"
+$CLICKHOUSE_CLIENT -q "CREATE VIEW x AS (SELECT $FUNCTION_NAME(COLUMNS(''))); -- { serverError BAD_ARGUMENTS }"
+$CLICKHOUSE_CLIENT -q "CREATE VIEW x AS (SELECT $FUNCTION_NAME(COLUMNS('t.*'))); -- { serverError BAD_ARGUMENTS }"
+$CLICKHOUSE_CLIENT -q "CREATE VIEW x AS (SELECT $FUNCTION_NAME(COLUMNS('t.t*')) FROM t); -- { serverError BAD_ARGUMENTS }"
```

Comment:
It's probably possible to support it with enabled analyzer, but i'm ok to forbit it.